mysql/toddl.go

```diff
@@ -229,3 +230,32 @@ func cvtForeignKeys(conv *internal.Conv, srcTable string, srcKeys []schema.Forei
 	}
 	return spKeys
 }
+
+func cvtIndexes(conv *internal.Conv, spTableName string, srcTable string, srcIndexes []schema.Index, spIndexNames map[string]bool) []ddl.CreateIndex {
```

Comment:
Suggest renaming: 
  spIndexNames -> usedNames
to better reflect what this parameter does (and for consistency with callsite).

Reply:
updated as mentioned